lib/auth.rb

```diff
@@ -6,9 +6,9 @@ module Auth; end
 require 'auth/result'
 require 'auth/authenticator'
 require 'auth/managed_authenticator'
-require 'auth/omniauth_strategies/discourse_google_oauth2'
+require 'auth/omni_auth_strategies/discourse_google_oauth2'
 require 'auth/facebook_authenticator'
 require 'auth/github_authenticator'
 require 'auth/twitter_authenticator'
-require 'auth/google_oauth2_authenticator'
+require 'auth/google_o_auth2_authenticator'
```

Comment:
I think we need to have some consistency here. `'auth/omni_auth_strategies/discourse_google_oauth2'` vs `'auth/google_o_auth2_authenticator'`. Is it `oauth2` or `o_auth2`? xD

Reply:
Well actually the idea was to not change the existing constants :sweat_smile: This is indeed simpler with inflections.

Reply:
ahh let us live with this for now since I do not think we should break constants as part of this upgrade.